jujugui/static/gui/src/app/init.js

```diff
@@ -27,6 +27,8 @@ window.juju.utils.RelationUtils = require('./init/relation-utils');
 const yui = window.yui;
 window.models = yui.namespace('juju.models');
 
+window.jujulib = require('./jujulib');
```

Comment:
This places the jujulib back on the window where the modules expect to find it. This will be redundant once the modules require() the jujulib sub-modules.